runtime/bindings/python/iree/runtime/array_interop.py

```diff
@@ -71,6 +73,7 @@ def __init__(
         self._buffer_view = buffer_view
         self._implicit_host_transfer = implicit_host_transfer
         self._override_dtype = override_dtype
+        self._cb = HalCommandBuffer(self._device)
```

Comment:
you can't reuse a command buffer like this (they are recorded once and can never be re-recorded, so if you tried to do two copies to host you'd explode) - use iree_hal_device_queue_copy instead